utils/text_formatting.jsx

```diff
@@ -319,8 +319,13 @@ function highlightCurrentMentions(text, tokens, mentionKeys = []) {
             flags += 'i';
         }
 
-        const pattern = new RegExp(`(^|\\W)(${escapeRegex(mention.key)})(\\b|_+\\b)`, flags);
-
+        let pattern;
+        if (cjkPattern.test(mention.key)) {
+            // In the case of CJK mention key, even if there no delimiters (such as spaces) at both ends of a word, it is recognized as a mention key
+            pattern = new RegExp(`(^|.)(${escapeRegex(mention.key)})($|.)`, flags);
```

Comment:
For the prefix and suffix groups, you can just use empty brackets like `()` to get the same results. I think the ones that you’re using may not match text right at the start or end of a line in all cases

Reply:
You're right! I've fix this.